start.html

```diff
@@ -355,7 +556,114 @@ <h3 class="accordion">
 			   can complete their installation of
 			   pbench-agent as described above. The public
 			   key should be added to the authorized-keys
-			   file of the Pbench user on the server where
+			   file of the relevant hosts and
+			   will copy all the results into the standard
+			   directory which postprocessing will copy
+			   over to the controller
+			   host. pbench-user-benchmark calls the
+			   script in its command-line arguments
+			   (everything after the – is just execed by
+			   -pbench-user-benchmark) and redirects its
+			   stdout to a file in that
+			   directory: <span class="c_snip_inline">$benchmark_run_dir/result.txt</span>.
+                         </p>
+			 <p class="info_text"  style="text-align:justify">
+                           Finally, you should provide an inventory
+                           file template (see above) with the the URLs
+                           for the config file and the ssh key file
+                           filled in for users to use.
+                         </p>
+                         <!-- Something is screwy here -->
+			 </div>
+		       </div>
+		     </div>
+		   </div>
+		 </div>
+	       </div>
+	     </div>
+	   </div>
+	   <div  id="best" class="tabcontent">
+	     <div class="App">
+	       <div class="section">
+		 <div class="pf-c-accordion accor_card">
+		   <div class="accordionCard">
+		     <h3 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="true">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text"> Clear results </span>
+		       </button>
+		     </h3>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" style="display: none;">
+		       <div class="pf-c-accordion__expanded-content-body">
+			 <p class="info_text" style="text-align:justify">
+			   The pbench-move-results script removes the
+			   results directory (assumed to be within the
```

Comment:
I think it now fetches the correct `${pbench_run}` directory from the `pbench-agent.cfg` file.

Reply:
This comment (and subsequent comments) refer to sections that are beyond the scope of this PR (although I did fix some things here and there opportunisticaly). If you don't mind, I'll open an issue and copy all these comments there to be addressed later.

Reply:
See issue #1927.